app/views/shared/_time_entry_input.html.slim

```diff
@@ -6,6 +6,8 @@
       flatpickr_minute_Increment: 1,
       flatpickr_enable_time: true,
       flatpickr_time_24hr: true,
+      flatpickr_default_hour: Time.now.hour,
+      flatpickr_default_minute: Time.now.min,
```

Comment:
Does this work even if the page has been running for a while? I'd guess this is just initialized on page load, right?

Reply:
https://github.com/renuo/redmine_tracky/pull/67 does it in JS

Reply:
Yes its only on page load. Should I look for a way that updates it constantly?